docs/admin/kube-proxy.md

```diff
@@ -1,3 +1,125 @@
-This file is autogenerated, but we've stopped checking such files into the
-repository to reduce the need for rebases. Please run hack/generate-docs.sh to
-populate this file.
+## kube-proxy
```

Comment:
These changes should not be committed

Reply:
Not sure why the update-generated-docs.sh script did that.  The verify- counter part was failing. Ok will remove.